### PR TITLE
Proofread

### PR DESCRIPTION
**Pull request by st10740**
* Try rewriting the sentence to make it smoother
* Correct the mistranslated Chinese for 'process'
* Translate the untranslated sentence

### CODE REVIEW

**Review by jserv — CHANGES_REQUESTED** [review]
Read https://cbea.ms/git-commit/ carefully and refine the git commit messages.

**Review by jserv — CHANGES_REQUESTED** [review]
Avoid writing "Update <filename>" as the subject of git commits. Instead, you can say "Proofread <subsection name>". The message body should consist of complete English sentences if you want to explain something.

File: commodity-hardware-today/ram-types/dynamic-ram.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <figcaption>圖 2.5：1-T 動態 RAM</figcaption>
 </figure>
 
-一個動態 RAM 的記憶單元在電容 $$ \mathbf{C} $$ 中保存其狀態。電晶體 $$ \mathbf{M} $$ 用以控制狀態的存取。為了讀取記憶單元的狀態，要提高存取線路 $$ \mathbf{AL} $$ 的電位；這要不使得電流流經資料線路（data line） $$ \mathbf{DL} $$、要不沒有，取決於電容中的電量。要寫到記憶單元中，則要適當地設置資料線路 $$ \mathbf{DL} $$，然後提高 $$ \mathbf{AL} $$ 的電位一段足以讓電容充電或放電的時間。
+一個動態 RAM 的記憶單元在電容 $$ \mathbf{C} $$ 中保存其狀態。電晶體 $$ \mathbf{M} $$ 用以控制狀態的存取。為了讀取記憶單元的狀態，要提高存取線路 $$ \mathbf{AL} $$ 的電位；這要不使得電流流經資料線路（data line） $$ \mathbf{DL} $$、要不沒有，取決於電容中的電量。要寫到記憶單元中，則要適當地設置資料線路 $$ \mathbf{DL} $$，然後將 $$ \mathbf{AL} $$ 的電位提高至足夠長的時間，以讓電容充電或放電的時間。
```
Comment (jserv) [inline commodity-hardware-today/ram-types/dynamic-ram.md]: Check the part "以讓電容充電或放電的時間".

File: virtual-memory/impact-of-virtualization.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <figcaption>圖 4.4：Xen 虛擬化模型</figcaption>
 </figure>
 
-在 Xen 的情況下（見圖 4.4），Xen VMM 即是這個軟體。不過 VMM 本身並不實作太多其它的硬體控制。不若在其它較早期的系統（以及首次釋出的 Xen VMM）上的 VMM，除了記憶體與處理器之外的硬體是由具有特權的 Dom0 域所控制的。目前，這基本上是與沒有特權的 DomU 系統核心相同的系統核心，而且 –– 就所關心的記憶體管理而言 –– 它們沒什麼區別。重要的是，VMM 將實體記憶體分發給了 Dom0 與 DomU 系統核心，其因而實作了普通的記憶體管理，就好像它們是直接執行在一個處理器上一樣。
+在 Xen 的情況下（見圖 4.4），Xen VMM 即是這個軟體。不過 VMM 本身並不實作太多其它的硬體控制。不若在其它較早期的系統（以及首次釋出的 Xen VMM）上的 VMM，除了記憶體與處理器之外的硬體是由具有特權的 Dom0 域所控制的。目前，這基本上是與沒有特權的 DomU 系統核心具有相同的系統核心，而且 –– 就所關心的記憶體管理而言 –– 它們沒什麼區別。重要的是，VMM 將實體記憶體分發給了 Dom0 與 DomU 系統核心，其因而實作了普通的記憶體管理，就好像它們是直接執行在一個處理器上一樣。
```
Comment (jserv) [inline virtual-memory/impact-of-virtualization.md]: Replace "這基本上是與沒有特權的 DomU 系統核心具有相同的系統核心" with "這跟運作於非特權模式的 DomU 具備相同的系統核心"

> Unlike VMMs on other, earlier systems (and the first release of the Xen VMM) the hardware outside of memory and processors is controlled by the privileged Dom0 domain. Currently, this is basically the same kernel as the unprivileged DomU kernels and, as far as memory handling is concerned, they do not differ. Important here is that the VMM hands out physical memory to the Dom0 and DomU kernels which, themselves, then implement the usual memory handling as if they were running directly on a processor.

File: virtual-memory/optimizing-page-table-access/influencing-tlb-performance.md [hunk 3]
```diff
@@ -24,6 +24,6 @@ Program Headers:
 使用較大的分頁容量有個次要的影響：分頁表樹的層級數量會被減低。由於對應到分頁偏移量的虛擬位址部分增加了，就沒有剩下那麼多需要透過分頁目錄處理的位元了。這表示，在一次 TLB 錯失的情況下，必須完成的工作總量減少了。
 
 除了使用大分頁尺寸外，也可能藉由將同時用到的資料搬移到較少的分頁上，以減少所需的 TLB 項目數量。這類似於我們先前討論的針對快取使用的一些最佳化。
-Only now the alignment required is large.
+只是現在所需的對齊尺寸更大。
```
Comment (jserv) [inline virtual-memory/optimizing-page-table-access/influencing-tlb-performance.md]: Refine this statement.

**Comment by st10740** [comment]
I revised the above issues.

**Comment by jserv** [comment]
> I revised the above issues.

No, you shall avoid placing filenames in the subject of git commits.

**Comment by jserv** [comment]
Instead of "Proofread multi-level-page-tables.md // I change the translation about two level 2 page directories needed
to make the sentence more fluent," you should denote the paragraph you have modified along with the motivations.

**Comment by st10740** [comment]
I replaced the filenames in the subject of git commits with simple description about the sentences I modified, and explained the reason why I modified them in the message body.

**Comment by jserv** [comment]
Thank @st10740 for contributing!

**Pull request merged**